package.json

```diff
@@ -0,0 +1,15 @@
+{
+  "dependencies": {
+    "esbuild": "^0.18.11",
+    "jquery": "^3.7.0",
+    "jquery-ujs": "^1.2.3",
+    "sass": "^1.63.6",
+    "selectize": "^0.12.6"
+  },
+  "scripts": {
+    "build-dev:js": "esbuild app/assets/javascripts/administrate/application.js --bundle --sourcemap --outdir=app/assets/builds/administrate --public-path=/assets/administrate",
+    "build:js": "yarn build-dev:js --minify",
+    "build:css": "sass ./app/assets/stylesheets/administrate/application.scss:./app/assets/builds/administrate/application.css ./app/assets/stylesheets/administrate-internal/docs.scss:./app/assets/builds/administrate-internal/docs.css --source-map --load-path=node_modules --style=compressed",
+    "build-dev:css": "yarn build:css --watch"
+  }
```

Comment:
@pablobm I love how you simplified this! I was thinking: we may be able to simplify this even further by removing `build-dev`s commands :thinking: 

We could publish the gem with un-minified JS and CSS and delegate the minification to app importing the gem. Because people's apps will include our compiled `.js` and `.css` files into their assets pipelines which would apply minification.